### DISCUSSION TITLE
Photopea Fonts and Google Fonts

### DISCUSSION THREAD

**Discussion by caleb-bright** [photopea/photopea#6637] [Q&A] [answered]
How often is the Photopea font library synced to Google Fonts?

**Comment by photopea** [comment]
We compose our font library using free fonts from different sources. Is there a specific free font which you like, which is not in our library?

**Reply by caleb-bright** [reply]
I was wondering if the fonts synced with PhotoPea automatically or if they are manually added.

**Reply by photopea** [reply, marked as answer]
No, we update the font database manually.

**Comment by caleb-bright** [comment]
Oh alright, thanks for the information. I was just wondering. This may be
marked resolved.
